avocado-setup.py

```diff
@@ -233,6 +233,8 @@ def get_repo(repo, basepath, install=False):
     """
     repo_name = repo.split('/')[-1].split('.')[0]
     repo_path = os.path.join(basepath, repo_name)
+    if os.path.isdir(repo_path) and ('-b ' or '--branch ' in repo):
```

Comment:
@sathnaga Are we not handling the case where this branch itself is cloned, and the second bootstrap will remove it anyways ?

Reply:
@narasimhan-v yep, that is handled already, if no repo folder, it will be cloned fresh in next steps, and more over, we can not remember the branch details of previous time and finding out after going in will be tedious, it is better this way, this is more of contingency plan, ideally we do not need two times bootstrap, unless until there needs a change in env.conf/any repo, so this should cover pretty much....

Reply:
okay, we can add that in a new commit if and when needed.